tests/components/geo_json_events/test_geo_location.py

```diff
@@ -73,7 +70,6 @@ async def test_entity_lifecycle(
         assert state is not None
         assert state.name == "Title 1"
         assert state.attributes == {
-            ATTR_EXTERNAL_ID: "1234",
```

Comment:
Why is the external ID deleted in the tests?

Reply:
Good catch - that should be added back now.